docs/antrea-proxy.md

```diff
@@ -253,6 +253,24 @@ data:
       proxyLoadBalancerIPs: false
 ```
 
+It's worth noting that AntreaProxy will bypass all the external `loadBalancerIP`s
+with the above configuration. After K8s v1.29, field `LoadBalancerIPMode` was
```

Comment:
With the above configuration, AntreaProxy will ignore all external `loadBalancerIP`s.

Reply:
Done